setup.sh

```diff
@@ -96,24 +92,25 @@ build_elektron=0
 # Build configuration passed to fakeroot
 build_configuration=""
 
-if [ $# -eq 0 ]; then
-	echo "This will build and install the complete RCPRG robot software stack with default build options."
-	echo "Use '$0 --help' to see the available options and their defualt values."
-	read -r -p "Continue [Y/n]?" response
-	# tolower
-	response=${response,,}
-	if [[ $response =~ ^(yes|y| ) ]] || [ -z $response ]; then
-		echo "Starting the build!"
-		build_configuration=" -g -e -o -f -v"
-		build_elektron=1
-		build_gazebo=1
-		build_orocos=1
-		build_fabric=1
-		build_velma=1
-	else
-		exit 0
-	fi
-fi
+# DO NOT USE DEFAULT ARGS!!!!11
+#if [ $# -eq 0 ]; then
```

Comment:
J/w, jeśli ma wylecieć, to niech wyleci zamiast być zakomentowanym, zawsze zostaje w VCSie